pkg/build/graph/analysis/bc.go

```diff
@@ -160,6 +237,44 @@ func FindPendingTags(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// validImageStreamImage will cycle through the imageStream.Status.Tags.[]TagEvent.DockerImageReference and  determine whether an image with the hexadecimal image id
```

Comment:
Interesting question/point..... so obviously the ImageStreamImage already has an edge to the ImageStream.  But are we saying leave that link, and add an edge to a sub-field to the ImageStream.Status.Tags entry (redundant perhaps) or remove the edge to the ImageStream and just have the edge go to the ImageStream.Status.Tags entry (not 100% sure we should remove access to the other parts of the ImageStream ... could prove useful)? 

I'll include this in the follow up issue (but will of course react to others' takes on this if they are posted here).